Snail/Variable.swift

```diff
@@ -34,12 +35,13 @@ public class Variable<T> {
     public func bind(to variable: Variable<T>) {
         variable.asObservable().subscribe(onNext: { [weak self] value in
             self?.value = value
-        })
+        }).add(to: disposer)
     }
 
     public func map<U>(_ transform: @escaping (T) -> U) -> Variable<U> {
```

Comment:
also think we can delete this function entirely since from what i've seen you should `asObservable` to use operators on variables.